### PR TITLE
Point out listenToStream usage with events in README

### PR DESCRIPTION
**Pull request by noahbetzen-wk**
## Motivation
  
In general we prefer `listenToStream` as opposed to `listen` with events, so I added a section of the README that describes how to do this.

## Changes
  
Simple README update with example.

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: README.md [hunk 1]
```diff
@@ -197,6 +197,17 @@ sampleModule.events.valueChanged.listen((newValue) {
 });
 ```
 
+In order to automatically clean up subscriptions if the module is disposed, you can make use of `listenToStream` instead of the buit-in `listen`. See [the section on disposal](#disposal) for more details.
```
Comment (georgelesica-wf) [inline README.md]: "buit-in" #nit

**Comment by georgelesica-wf** [comment]
+1 with a nit

**Comment by georgelesica-wf** [comment]
+1

**Comment by noahbetzen-wk** [comment]
@Workiva/release-management-p

**Comment by georgelesica-wf** [comment]
QA +1 docs only change

**Comment by georgelesica-wf** [comment]
@Workiva/release-management-p

**Pull request merged**